src/ext/oblivc/obliv_bits.c

```diff
@@ -364,24 +364,22 @@ void protocolAddSizeCheck(ProtocolDesc* pd)
   t->cb.recv=sizeCheckRecv;
   t->cb.cleanup=sizeCheckCleanup;
 }
-// ---------------------------------------------------------------------------
+
+// --------------------------- Protocols -----------------------------------
```

Comment:
I'm assuming these are just shuffling things around, and not real code changes. Didn't look too closely.